src/ui-kit/Divider/Divider.module.scss

```
@@ -0,0 +1,9 @@
+.divider {
+  border-bottom: rem(1px) solid $grey-medium;
+  height: 1;
+  width: 100%;
+
+  &.divider-medium {
```

```suggestion
  &-medium {
```
il n'est pas nécéssaire d'augmenter la spécificité du selecteur dans ce cas, puisqu'il n'y a pas de surcharge.

---

C'est juste pour le ranger, c'est une variante du parent.